modules/beamdyn/src/Registry_BeamDyn.txt

```diff
@@ -44,6 +44,7 @@ typedef   ^        InitOutputType CHARACTER(ChanLen)  WriteOutputUnt {:}  -   -
 typedef   ^        InitOutputType ProgDesc            Ver             -   -   -   "This module's name, version, and date"  -
 typedef   ^        InitOutputType R8Ki                kp_coordinate  {:}{:} - -   "Key point coordinates array"            -
 typedef   ^        InitOutputType IntKi               kp_total        -   -   -   "Total number of key points" -
+typedef   ^        InitOutputType R8Ki                QPtN           {:}   -   -   "Quadrature (QuadPt) point locations in natural frame [-1, 1]" -
```

Comment:
Is there a reason that a part of the code outside of the BeamDyn module would need the BD quadrature points?  Also wondering why `kp_coordinates` are InitOutputType (I realize that is an old issue and not part of this PR.)

Reply:
I think this was mistakenly kept during the merge. It shouldn't be there as it isn't used anywhere else.  Thanks for catching this!

Reply:
Removed!